### PR TITLE
issue-5367 grabFromDatabase() returns string

### LINKED ISSUE #5367: Fix return type of grabFromDatabase() method to string | bool
grabFromDatabase() sometimes returns string. That will be more honest and will help with IDE notices.

### PR DESCRIPTION
**Pull request by SanzhiyevMergen**
Fixes #5367

### CODE REVIEW

File: src/Codeception/Module/Db.php [hunk 1]
```diff
@@ -869,7 +869,7 @@ public function grabColumnFromDatabase($table, $column, array $criteria = [])
      * @param string $column
      * @param array  $criteria
      *
-     * @return array
+     * @return string|bool
```
Comment (Naktibalda) [inline src/Codeception/Module/Db.php]: This method can return other types too, so it would be the best to change it to `@return mixed`

Comment (SanzhiyevMergen) [reply src/Codeception/Module/Db.php]: Ok, at least `null` can be returned. Agree.

Comment (SanzhiyevMergen) [reply src/Codeception/Module/Db.php]: Done.

Comment (Naktibalda) [reply src/Codeception/Module/Db.php]: integer too and probably float, because some database libraries return native types.

**Pull request merged**